gcf/gcf-ai-platform-example/main.py

```diff
@@ -0,0 +1,97 @@
+import json
+
+from flask import current_app as app
+
+import googleapiclient.discovery
+
+
+def _generate_payload(sentence):
```

Comment:
nit: Add Args and returns. 
Some functions use Function documentation (3.8.3 Functions and Methods)
https://github.com/google/styleguide/blob/gh-pages/pyguide.md

Reply:
doc string added